lib/internal/bootstrap_node.js

```diff
@@ -196,7 +200,12 @@
       enumerable: false,
       configurable: true
     });
-    global.process = process;
+    Object.defineProperty(global, 'process', {
```

Comment:
It was requested by @sam-github in https://github.com/nodejs/node/pull/10135#issuecomment-265188209.